examples/acis/Mapping_Example.py

```diff
@@ -0,0 +1,149 @@
+# review our data.
+
+myData = acis_request(method, parameters)
+
+print(myData)
+
+###########################################
+# MultiStnData calls take longer to return than single stations, especially when
+# you request multiple states. We can see the data is divided by station, with
+# each station having it's own meta and data components. This time we also have
+# multiple values in each data list. Each value corresponds to the variable we
+# requested, in the order we requested it. So in this case, we have the
+# precipitation value, followed by the departure from normal value. Before we
+# plot this information, we need to add up the precipitation sums. But rather
+# than doing it in Python, let's make another ACIS call that prepares this for
+# us.
+
+parameters = {'state': 'co', 'sdate': '20130909', 'edate': '20130912', 'elems': [
+    {'name': 'pcpn', 'interval': 'dly', 'smry': 'sum', 'smry_only': 1},
+    {'name': 'pcpn', 'interval': 'dly', 'smry': 'sum', 'smry_only': 1, 'normal': 'departure'}]}
+
+myData = acis_request(method, parameters)
+
+print(myData)
+
+###########################################
+# First of all, we have two new components to our elements: 'smry' and 'smry_only'.
+# 'smry' allows us to summarize the data over our time period. There are a few
+# options for this, including being able to count the number of records exceeding
+# a threshold (something we will explore in the next example). The other parameter,
+# 'smry_only', allows us to only return the summary value and not the intermediate
+# data.
+#
+# Now let's look at how our data has changed. Rather than having a just a 'meta'
+# and 'data' component, we have a new one called 'smry'. As you've guessed,
+# this contains our summary information (also in the order we requested it).
+# By specifying 'smry_only', there is no 'data' component. If we also wanted
+# all 4 days of data, we would simply remove that parameter.
+#
+# To wrap up this example, we will finally plot our precipitation sums and
+# departures onto a map using CartoPy. To do this we will utilize
+# the meta data that is provided with each station's data. Within the metadata
+# is a 'll' element that contains the latitude and longitude, which is perfect
+# for plotting!
+#
+# One final thing to note is that not all stations have location information.
+# Stations from the ThreadEx network cover general areas, and thus aren't
+# packaged with precise latitudes and longitudes. We will skip them by
+# identifying their network ID of 9 in the ACIS metadata. Don't worry about
+# lost information though! These summarize stations that already exist within
+# their areas!
+
+lat = []
+lon = []
+pcpn = []
+pcpn_dep = []
+
+for stn in myData['data']:
+    # Skip threaded stations! They have no lat/lons
+    if stn['meta']['sids'][-1][-1] == '9':
+        continue
+    # Skip stations with missing data
+    if stn['smry'][0] == 'M' or stn['smry'][1] == 'M':
```

Comment:
What comes before “M” if it’s at index 1? White space? If so, what about:
```
If stn[‘smry’].lstrip().startswith(‘M’)
```
?

Reply:
In this case 'smry' is actually a list of summary values. So in this case the 'smry' would have one value for the total precip and another for the departure from normal.